cloudinit/subp.py

```diff
@@ -144,6 +144,23 @@ def _indent_text(
         return text.rstrip(b"\n").replace(b"\n", b"\n" + b" " * indent_level)
 
 
+def is_invalid_command(args: Union[List[str], List[bytes]]):
```

Comment:
Nit: I think something like `raise_on_invalid_command` would be a little more clear.